go/x509/x509.go

```diff
@@ -1123,12 +1228,13 @@ func parseCertificate(in *certificate) (*Certificate, error) {
 					out.IssuingCertificateURL = append(out.IssuingCertificateURL, string(v.Location.Bytes))
 				}
 			}
+		} else {
+			// Unknown extensions are recorded if critical.
+			unhandled = true
 		}
 
-		if e.Critical {
-			// START CT CHANGES
-			nfe.AddError(UnhandledCriticalExtension{e.Id})
-			// END CT CHANGES
+		if e.Critical && unhandled {
```

Comment:
Looks like the `// (START|END) CT CHANGES` wrapper got merged out here, was that intentional?